packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -335,7 +335,7 @@ function computronCounter(
     return totalBeans === 0n && !cleanupDone;
   };
   const didCleanup = details => {
-    for (const [phase, count] of Object.entries(details)) {
+    for (const [phase, count] of Object.entries(details.cleanups)) {
```

Comment:
Wondering if types could somehow have helped discover this one.

Reply:
Yes, but I've currently got them reserved for a followup.